src/palace/manager/core/monitor.py

```diff
@@ -505,12 +514,31 @@ def run_once(self, *ignore):
         # update.
         return TimestampData(counter=offset, achievements=achievements)
 
+    @retry(
+        retry=(
+            retry_if_exception_type(StaleDataError)
+            | retry_if_exception_type(ObjectDeletedError)
+        ),
+        stop=stop_after_attempt(MAXIMUM_BATCH_RETRIES),
+        wait=wait_exponential(multiplier=1, min=1, max=60),
+        reraise=True,
+    )
```

Comment:
Why add this code to both the monitor case class and to the axis monitor directly?

Reply:
Not sure I understand the question.   If I understand it the reason was that they don't share the same "process" methods.  SweepMonitor and Axis360CirculationMonitor share a common ancestor (CollectionMonitor) but the processing is handled by subclasses.